tests/integrations/test_dsm.py

```diff
@@ -12,7 +12,7 @@ class Test_DsmKafka:
     """ Verify DSM stats points for Kafka """
 
     def setup_dsm_kafka(self):
-        self.r = weblog.get("/dsm?integration=kafka")
+        self.r = weblog.get("/dsm?integration=kafka", timeout=60)
```

Comment:
Can we make this a kwarg with a default of 60, and/or recreate as `weblog.get_with_timeout()` (also with the default)

Reply:
`weblog.get` already accepts timeout as a kwarg (default is 5 seconds), which is why I am using a kwarg here

Reply:
Do not change the default value, if for some reason the weblog is broken, and do not respond, it'll make the CI run for hours before we see an error, as responses are not checked during the setup phase. 

Though, I'm really surprised to see that a 60s timeout is needed for kafka, do we know why it's so slow? Is there something that can be done to make it answers faster?

Reply:
I've removed the timeouts